.github/workflows/feature.yml

```diff
@@ -5,15 +5,13 @@ on:
     branches:
       - feature/**
       - bug/**
-      - bugfix/**
```

Comment:
Can we add these back in?

Reply:
This came from one of my develop merges, not sure what it's about myself...

Reply:
Reverted :)